client/aliases.go

```diff
@@ -40,31 +41,21 @@ func (client *Client) performAliasAction(sa *aliasAction) (changeID string, err
 	return client.doAsync("POST", "/v2/aliases", nil, nil, bytes.NewReader(b))
 }
 
-// Alias enables the provided aliases for the snap with snapName.
-func (client *Client) Alias(snapName string, aliases []string) (changeID string, err error) {
+// Alias sets up a manual alias from alias to app in snapName.
+func (client *Client) Alias(snapName, app, alias string) (changeID string, err error) {
 	return client.performAliasAction(&aliasAction{
-		Action:  "alias",
-		Snap:    snapName,
-		Aliases: aliases,
+		Action: "alias",
+		Snap:   snapName,
+		Target: app,
+		Alias:  alias,
 	})
 }
 
-// Unalias disables explicitly the provided aliases for the snap with snapName.
-func (client *Client) Unalias(snapName string, aliases []string) (changeID string, err error) {
+// Unalias tears down a manual alias or disables all aliases of a snap (removing all manual ones)
```

Comment:
@niemeyer done something along the lines of splitting but kept just "snap unalias" as discussed, introduced both RemoveManualAlias and DisableAllAliases (it actually touches also manual aliases to be consistent with other disabling cases)